core/Cargo.toml

```diff
@@ -39,3 +44,4 @@ rkyv-impl = [
     "rkyv",
     "bytecheck"
 ]
+serde = ["dep:serde", "dusk-bls12_381/serde", "bs58", "base64", "hex", "serde_json", "dusk-jubjub/serde"]
```

Comment:
please don't use `dusk-bls12_381/serde` because it doesn't use the serde serialization we want (and changing it would result in a major version bump).
@Neotamandua has a workaround for serializing the `BlsScalar`